cypress/integration/login.js

```diff
@@ -1,94 +1,115 @@
+        loginPage.getSpecificRoleFromDropDown("Manager").click();
+        loginPage.getSubmitButton().click();
+        loginPage.getUserInfoTitle("Edvinas Kveselys");
+        loginPage.getNavigationLength().should('have.length', 5);
+        loginPage.getActiveNavLinkColor("rgb(64, 76, 237)");
+        loginPage.getActiveNavLink("/time-logging");
     })
+
     it('Verify "Accountant" can log in and should see appropriate tabs', function() {
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Edvinas Kveselys"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="Accountant"]').click();
-        cy.get('[type="submit"]').click();
-        cy.get('.user-info__title').contains('Edvinas Kveselys');
-        cy.get('.main-nav').find('li').should('have.length', 5);
-        cy.get('.main-nav__link--active').should('have.css', 'color').and('eq', 'rgb(64, 76, 237)');
-        cy.get('.main-nav__link--active').should('have.attr', 'href', '/time-logging');
+        loginPage.visit();
+        loginPage.getLoginUserId().click({force:true});
+        loginPage.getSpecificUserFromDropDown("Edvinas Kveselys").click();
+        loginPage.getLoginRole().click({force:true});
+        loginPage.getSpecificRoleFromDropDown("Accountant").click();
+        loginPage.getSubmitButton().click();
+        loginPage.getUserInfoTitle("Edvinas Kveselys");
+        loginPage.getNavigationLength().should('have.length', 5);
+        loginPage.getActiveNavLinkColor("rgb(64, 76, 237)");
+        loginPage.getActiveNavLink("/time-logging");
     })
+
     it('Verify "Admin" can log in and should see appropriate tabs', function() {
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Edvinas Kveselys"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="Admin"]').click();
-        cy.get('[type="submit"]').click();
-        cy.get('.user-info__title').contains('Edvinas Kveselys');
-        cy.get('.main-nav').find('li').should('have.length', 6);
-        cy.get('.main-nav__link--active').should('have.css', 'color').and('eq', 'rgb(64, 76, 237)');
-        cy.get('.main-nav__link--active').should('have.attr', 'href', '/time-logging');
+        loginPage.visit();
+        loginPage.getLoginUserId().click({force:true});
+        loginPage.getSpecificUserFromDropDown("Edvinas Kveselys").click();
+        loginPage.getLoginRole().click({force:true});
+        loginPage.getSpecificRoleFromDropDown("Admin").click();
+        loginPage.getSubmitButton().click();
+        loginPage.getUserInfoTitle("Edvinas Kveselys");
+        loginPage.getNavigationLength().should('have.length', 6);
+        loginPage.getActiveNavLinkColor("rgb(64, 76, 237)");
+        loginPage.getActiveNavLink("/time-logging");
     })
+
+    /*it('Admin creates new task', function() {

```

Comment:
Never commint commented out code, if it is commented chances are it should be deleted :)